saleor/order/fetch.py

```diff
@@ -86,16 +82,7 @@ class EditableOrderLineInfo(LineInfo):
 
     @cached_property
     def variant_discounted_price(self) -> Money:
-        """Return the discounted variant price.
```

Comment:
Actually, after this change, I am not sure if we still need `variant_discounted_price`. 
With previous logic, the main purpose of this property was: return the price from listing if present. If not present, calculated it. 
But with your changes, the `order_line.base_unit_price` will be static, for the whole process for single line. So I think, that we could drop this logic 🤔 
(Correct me if I'm am wrong)